jobs/build-microk8s-charm.yaml

```diff
@@ -49,15 +54,31 @@
           [[ -f $WORKSPACE/.env ]] && source $WORKSPACE/.env
           set +o allexport
 
+          container="${JOB_NAME}-${BUILD_NUMBER}"
```

Comment:
If the job does not finish cleanly or gets stuck and we have to kill it, would we be leaking containers in this way?

Reply:
Added an initial step that cleans up any left-over containers from previous runs